src/mercury_engine_data_structures/cli.py

```diff
@@ -177,7 +177,7 @@ def decode_encode_compare_file(file_path: Path, game: Game, file_format: str):
         resource = resource_class.parse(raw, target_game=game)
         encoded = resource.build()
 
-        if raw != encoded and raw.rstrip(b"\xFF") != encoded:
+        if raw != encoded and raw.rstrip(b"\xff") != encoded:
```

Comment:
Personal preference but hex escapes look better with A-F capitalized, especially \x.

Reply:
I'm indifferent, but I don't believe it's configurable